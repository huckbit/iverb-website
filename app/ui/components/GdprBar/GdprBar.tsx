'use client';
import CookieConsent from 'react-cookie-consent';
import Link from 'next/link';

export default function GdprBar() {
  return (
    <CookieConsent location='bottom' buttonText='Accept' cookieName='gdpr' style={{ background: '#8a7852' }} buttonStyle={{ background: '#3d5371', color: '#fff', fontSize: '13px' }} expires={150}>
      This website uses cookies to enhance the user experience. By using this website, you agree to the use of cookies.{' '}
      <Link href='https://www.cookiesandyou.com/' target='_blank' rel='noopener noreferrer'>
        Learn more
      </Link>
    </CookieConsent>
  );
}
