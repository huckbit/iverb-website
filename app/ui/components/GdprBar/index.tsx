export { default as GdprBar } from './GdprBar';
