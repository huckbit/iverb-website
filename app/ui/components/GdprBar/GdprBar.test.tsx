import { render } from '@testing-library/react';
import GdprBar from './GdprBar';

test('matches snapshot', () => {
  const container = render(<GdprBar />);
  expect(container).toMatchSnapshot();
});
