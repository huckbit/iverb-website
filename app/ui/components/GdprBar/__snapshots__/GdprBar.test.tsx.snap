// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="CookieConsent"
        style="align-items: baseline; background: rgb(138, 120, 82); color: white; display: flex; flex-wrap: wrap; justify-content: space-between; left: 0px; position: fixed; width: 100%; z-index: 999; bottom: 0px;"
      >
        <div
          class=""
          style="flex: 1 0 300px; margin: 15px;"
        >
          This website uses cookies to enhance the user experience. By using this website, you agree to the use of cookies.
           
          <a
            href="https://www.cookiesandyou.com/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Learn more
          </a>
        </div>
        <div
          class=""
        >
          <button
            aria-label="Accept cookies"
            class=""
            id="rcc-confirm-button"
            style="background: rgb(61, 83, 113); border: 0px; border-radius: 0px; box-shadow: none; color: rgb(255, 255, 255); cursor: pointer; flex: 0 0 auto; padding: 5px 10px; margin: 15px; font-size: 13px;"
          >
            Accept
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="CookieConsent"
      style="align-items: baseline; background: rgb(138, 120, 82); color: white; display: flex; flex-wrap: wrap; justify-content: space-between; left: 0px; position: fixed; width: 100%; z-index: 999; bottom: 0px;"
    >
      <div
        class=""
        style="flex: 1 0 300px; margin: 15px;"
      >
        This website uses cookies to enhance the user experience. By using this website, you agree to the use of cookies.
         
        <a
          href="https://www.cookiesandyou.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Learn more
        </a>
      </div>
      <div
        class=""
      >
        <button
          aria-label="Accept cookies"
          class=""
          id="rcc-confirm-button"
          style="background: rgb(61, 83, 113); border: 0px; border-radius: 0px; box-shadow: none; color: rgb(255, 255, 255); cursor: pointer; flex: 0 0 auto; padding: 5px 10px; margin: 15px; font-size: 13px;"
        >
          Accept
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
